packages/vuetify/src/components/VDataTable/VDataTable.sass

```diff
@@ -164,3 +164,6 @@
 
       &-active
         color: $data-table-header-mobile-chip-icon-color-active
+
+.v-data-table__mask
+  background: rgb(var(--v-theme-on-surface-variant))
```

Comment:
Is there a way we can make this more flexible?

`<v-data-table :headers="headers" :items="items" :search="search" class="bg-primary" item-value="name" />`

![image](https://github.com/user-attachments/assets/dfdf1707-8a03-4ad7-8d95-c0ac407146a0)

Reply:
Yes it fixes it but I hate that fix. Mainly because when we say theme="dark", we're saying use those colors. So now everything inside of that component would operate with a different palette.